src/drivers/extjs/components/dataview.js

```diff
@@ -4,14 +4,14 @@ import {HTMLComponentBase} from '../../html/components/base.js'
 import {ExtJsComponentBase} from './base.js'
 
 export class ExtJsComponentDataView extends ExtJsComponentBase {
-    select (callback, index, itemCls) {
+    select (callback, index) {
```

Comment:
Let's swap arguments to `select (index, callback)` like in other `select` methods in library